packages/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -606,6 +606,18 @@ export const oneSignalConfig: IConfigCredentials[] = [
     type: 'text',
     required: true,
   },
+  {
+    key: CredentialsKeyEnum.Region,
```

Comment:
@suplere do we need to use `CredentialsKeyEnum.Domain` here as we use `this.apiVersion = config.domain;` in the provider?

Reply:
Yes, I have updated this to be apiVersion and passed the proper info from the handler